src/components/templates/TopBar.jsx

```diff
@@ -9,18 +9,18 @@ const TopBar = () => {
           <ListItem
             icon="/images/location.svg"
             text="90919 Madie run Apt. 790"
-            textStyle="text-white"
+            textStyle="text-white sm:text-lg text-base"
           />
           <ListItem
             icon="/images/ion-mail-outline.svg"
             text="medico@health.care"
-            textStyle="text-white"
+            textStyle="text-white sm:text-lg text-base"
           />
           <ListItem
             icon="/images/whatsapp.svg"
             text="Connect on Whatsapp"
             styles="md:ml-auto"
-            textStyle="underline underline-offset-4 text-white"
+            textStyle="underline underline-offset-4 text-white sm:text-lg text-base"
```

Comment:
We are using the same text size over and over, so instead of adding it multiple times, add it to the base component (ListItem). For other variations you can add custom one.